src/utils/update/psc.to.beneficial.owner.type.mapper.ts

```diff
@@ -0,0 +1,173 @@
+import { OfficeAddress } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
+import { CompanyPersonWithSignificantControl } from "@companieshouse/api-sdk-node/dist/services/company-psc/types";
+import { BeneficialOwnerGov } from "../../model/beneficial.owner.gov.model";
+import { BeneficialOwnerIndividual } from "../../model/beneficial.owner.individual.model";
+import { BeneficialOwnerOther } from "../../model/beneficial.owner.other.model";
+import { Address, InputDate, NatureOfControlType, yesNoResponse } from "../../model/data.types.model";
+
+export const mapPscToBeneficialOwnerTypeIndividual = (psc: CompanyPersonWithSignificantControl): BeneficialOwnerIndividual => {
```

Comment:
I think maybe psc should be of type `CompanyPersonWithSignificantControlResource` to match up with the service's item list of `CompanyPersonWithSignificantControlResource`'s? We had a look at this and were multiple similar names just to be awkward 😆

Reply:
the service is: 
public async getCompanyPsc (number: string): Promise<Resource<CompanyPersonsWithSignificantControl>> {
the Resource has the snake case fields - here it's camel case / not Resource

Reply:
Now using CompanyPersonWithSignificantControlResource